### ISSUE TITLE
Almost there... on the new timer filter, but

### ISSUE THREAD

**Issue by CKMacLeod** [ronalfy/simple-comment-editing#10]
Getting a "see-through" display of the "{minutes_time}" variable when it goes < 1.
![sce_minutes_time_variable_printing](https://cloud.githubusercontent.com/assets/6822649/9924392/d5c44a6a-5cb3-11e5-9f93-bcda362c6783.png)

using 1.61, sce_timer filter as supplied

**Comment by ronalfy** [comment]
Can you please check now with 1.65?

```php
add_filter( 'sce_timer_output', function( $string ) {
    return "{minutes_time}{colon}{seconds_time}"; //format minute:seconds
}, 11, 1 );
```

**Comment by ronalfy** [comment]
If this doesn't work, I'm reverting the changes and going back to the drawing board.

**Comment by CKMacLeod** [comment]
It works! 

When I understand the code better, I may look into a creating a slightly different result, and I wonder how adjustable this will be for people who want some variation - e.g., 00.59, or 0 mins, 59 secs or 0'59" etc., - or who want it to change color or animate, or who want to employ some kind of clock or other image - but thanks for attending to this, and I hope that others also find it a useful enhancement.

Playing with it, I found that this also works, and preserves the colon once the minutes are in the zeroes

`add_filter( 'sce_timer_output', function( $string ) {
    return "{minutes_time}" . ":" . "{seconds_time}"; //format minute:seconds
}, 11, 1 );`

I was not able on a first experiment to produce a conditional relating to the `{minutes_time}` variable, but I'm just hacking around at the moment without much knowledge. When I look at the code that produces it, I may have a better idea, but I also figured that first priority for you would be putting out good code in the plugin, and I think you've got it. I'll let you know if I run into any other unexpected behavior.
